virttest/utils_net.py

```diff
@@ -1697,20 +1697,30 @@ def restart_guest_network(
     :param timeout: timeout value for command.
     """
     if os_type == "linux":
+        dhcp_clients = [("dhclient", "-r"), ("dhcpcd", "-k")]
```

Comment:
It's okay for me here, but for a basic system, both `dhcpcd` and `dhclient` are not mandatory as NetworkManager can also manage DHCP.

Reply:
In the NetworkManager service, which depends on .nmconnection files created during VM installation, no corresponding .nmconnection files exist when the VM boots up with multi-nics. so, NetworkManager cannot be used in such situation.

Reply:
`nmcli device up <eth_name>` will create the connection and get IP.

Reply:
It seems that nmcli could do a similar function, but I still prefer to keep the original design to maintain the stability of the framework